store/errors/errors.go

```diff
@@ -0,0 +1,19 @@
+package errors
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type Error = sdk.Error
+
+func ErrInternal(msg string) Error {
```

Comment:
Why not just use the errors in the `types/` package?

Reply:
I think it is, this is just a wrapper